NSS-Likeable-Lemurs/BangazonSite
# First task

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonSite/issues/32)

## Description of Proposed Changes
- Made it so that "sell a product" in the navbar only appears once the user is signed in.

## Steps to Test

```sh
git fetch --all
git checkout <firstTask>
launch application, sign in, and make sure that "sell a product" appears when you are signed in and not there when you are not
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* _Layout.cshtml and _LoginPartial

## Mentions @username

@kellycook301 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
